test/fuzz/common/sqrt.t.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.19 <0.9.0;
+
+import { sqrt } from "src/Common.sol";
+
+import { Base_Test } from "../../Base.t.sol";
+
+/// @dev Collection of tests for the square root function available in `Common.sol`.
+contract Common_Sqrt_Test is Base_Test {
+    uint256 internal constant MAX_SQRT = type(uint128).max;
+
+    function testFuzz_Sqrt_Of256BitNumber(uint256 x) external pure {
+        vm.assertLe(sqrt(x) ** 2, x, "Common sqrt");
+        if (x < MAX_SQRT ** 2) {
+            vm.assertGt((sqrt(x) + 1) ** 2, x, "Common sqrt");
+        } else {
+            // This case cannot be tested nicely using 'vm.assume()',
+            // because the fuzzer fails to find enough valid input.
+            // That's why we have to embed it here with an 'if'.
+            vm.assertEq(sqrt(x), MAX_SQRT, "Common sqrt");
```

Comment:
I''ll move this to another text function and use [`bound`](https://getfoundry.sh/reference/cheatcodes/assume#description):

> For broad checks, such as ensuring a uint256 falls within a certain range, you can bound your input with the modulo operator or Forge Standard's bound method.